integration/client/export_test.go

```diff
@@ -250,7 +250,7 @@ func TestExportAllCases(t *testing.T) {
 
 			img := tc.prepare(ctx, t, client)
 
-			dstFile, err := os.CreateTemp("", "export-test")
+			dstFile, err := os.CreateTemp(t.TempDir(), "export-test")
 			if err != nil {
 				t.Fatal(err)
 			}
```

Comment:
(the actual line is too far outside the diff for GitHub to let me comment)

Since switching to `t.TempDir` we do not need an explicit `defer` for cleanup.  Please remove the `defer` lines 257-260.